### PR TITLE
[SD-4293] Cannot upload image to avatar

### PR DESCRIPTION
**Pull request by jerome-poisson**
Fixed avatar upload, the issue was due to the use of ng-file-select
instead of ngf-select, probably an API change in ng-file-upload

SD-4293

### CODE REVIEW

**Comment by sjunaid** [comment]
:+1: (can you check the same for `ng-file-drop` as well)

File: scripts/superdesk-users/views/change-avatar.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h3 class="pull-left">{{ activity.label|translate }}</h3>
             <div class="dropzone" ng-file-drop="preview.img = $files[0]">
```
Comment (petrjasek) [inline scripts/superdesk-users/views/change-avatar.html]: seems like here it will need the same change, right @sjunaid ?

Comment (jerome-poisson) [inline scripts/superdesk-users/views/change-avatar.html]: yes indeed the drop was also bugged, fixed it

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**